src/server/codegen/knowledgepack_codegen_mixin.py

```diff
@@ -233,6 +232,9 @@ def create_sml_classification_result_info(self, models_data):
         model_fill["TensorFlow Lite for Microcontrollers"] = (
             "{\n\ttf_micro_model_results_object(kb_models[model_index].classifier_id, (model_results_t *)model_results);\n}"
         )
+        model_fill["Neural Network"] = (
```

Comment:
This needs an if statement that switchest between the two based on the inference engine selected